Microsoft.Maker.win10/Microsoft.Maker.Serial/Microsoft.Maker.Serial.vcxproj.filters

```diff
@@ -10,11 +10,13 @@
     <ClCompile Include="pch.cpp" />
     <ClCompile Include="..\..\source\Serial\BluetoothSerial.cpp" />
     <ClCompile Include="..\..\source\Serial\USBSerial.cpp" />
+    <ClCompile Include="..\..\source\Serial\DfRobotBleSerial.cpp" />
```

Comment:
Good call! We did talk with him face-to-face about what we were planning do to, but we didn't cover specifics. I'll make sure and have Paul ask him tomorrow, before we pull this in.